martin/src/pg/pool.rs

```diff
@@ -47,7 +47,7 @@ impl PgPool {
             .build()
             .map_err(|e| PostgresPoolBuildError(e, id.clone()))?;
 
-        let version = get_conn(&pool, id.as_str())
+        let version: String = get_conn(&pool, id.as_str())
```

Comment:
Seems there's few connection between cleaning up on `pg/pool.rs` and the target of this PR. Not sure as I didn't do much on mbtiles part of martin.

Reply:
sorry, that was a minor cleanup. Fixed